cardano-api/src/Cardano/Api/KeysShelley.hs

```diff
@@ -1188,7 +1188,9 @@ instance SerialiseAsBech32 (SigningKey StakePoolKey) where
     bech32PrefixesPermitted _ = ["pool_sk"]
 
 newtype instance Hash StakePoolKey =
-    StakePoolKeyHash (Shelley.KeyHash Shelley.StakePool StandardCrypto)
+    StakePoolKeyHash
```

Comment:
The record could exist on the same line as the constructor.

Reply:
Done.